.circleci/config.yml

```diff
@@ -80,6 +80,26 @@ jobs:
       - image: cimg/node:16.0.0
     steps:
       - common_test_steps
+  
+  GraphQL v15 Support:
+    docker:
+      - image: cimg/node:16.0.0
+    steps:
+      - oss/install_specific_npm_version:
+          version: '7'
+      - checkout
+      - run:
+          command: npm i -D graphql@15.8.0
+      - run:
+          command: npm run clean && npm i
+      - run:
+          command: npm run test:ci
+          environment:
+            JEST_JUNIT_OUTPUT_DIR: "reports/junit/"
+      - store_test_results:
+          path: reports/junit
+      - store_artifacts:
+          path: reports/junit
```

Comment:
@abernix I'd be curious to hear if you have any opinions on this. We've run into a couple v15 compat issues, this particular version straddle has a few gotchas. Seems like a reasonable way to catch this in CI.

I found the  `npm run clean && npm i` to workaround a problem where it seemed the TS build was out of date (probably was via the caching, which this breaks/works around). Maybe there's a better way that's still cacheable?